File: c/PRAK205-2510817210027-Ahmad_Tibrizi.c
#include <stdio.h>
#include <math.h>

int main() {
    float A, B, alas, tinggi, keliling, luas;

    printf("Masukkan nilai A: ");
    scanf("%f", &A);
    printf("Masukkan nilai B: ");
    scanf("%f", &B);

    alas = sqrt(B*B - A*A);
    tinggi = A;
    keliling = alas + tinggi + B;
    luas = (alas * tinggi) / 2;

    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n", luas);

    return 0;
}

File: python/PRAK205-2510817210027-Ahmad_Tibrizi.py
import math

A = int(input("Masukkan nilai A: "))
B = int(input("Masukkan nilai B: "))

alas = math.sqrt(B**2 - A**2)
tinggi = A
keliling = alas + tinggi + B
luas = (alas * tinggi) / 2

print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")